icons/router.svg

```diff
@@ -0,0 +1,26 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_1_3)" filter="url(#filter0_d_1_3)">
+<rect width="24" height="24" transform="translate(4)" fill="white"/>
+<path d="M6 20V16C6 14.8954 6.89543 14 8 14H24C25.1046 14 26 14.8954 26 16V20C26 21.1046 25.1046 22 24 22H8C6.89543 22 6 21.1046 6 20Z" stroke="black" stroke-width="2"/>
+<path d="M14 18C14.0003 18 14.0013 18.0001 14.0009 18.0001" stroke="black" stroke-width="2" stroke-linecap="round"/>
+<path d="M10 18C10.0003 18 10.0013 18.0001 10.0009 18.0001" stroke="black" stroke-width="2" stroke-linecap="round"/>
+<path d="M19 13V10" stroke="black" stroke-width="2" stroke-linecap="round"/>
+<path d="M16 7.16V7.16C17.712 5.64205 20.288 5.64205 22 7.16V7.16" stroke="black" stroke-width="2" stroke-linecap="round"/>
+<path d="M13.5 4.5V4.5C16.4571 1.27406 21.5429 1.27406 24.5 4.5V4.5" stroke="black" stroke-width="2" stroke-linecap="round"/>
+</g>
+<defs>
+<filter id="filter0_d_1_3" x="0" y="0" width="32" height="32" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
+<feFlood flood-opacity="0" result="BackgroundImageFix"/>
+<feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/>
+<feOffset dy="4"/>
+<feGaussianBlur stdDeviation="2"/>
+<feComposite in2="hardAlpha" operator="out"/>
+<feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.25 0"/>
+<feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow_1_3"/>
+<feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow_1_3" result="shape"/>
+</filter>
+<clipPath id="clip0_1_3">
+<rect width="24" height="24" fill="white" transform="translate(4)"/>
+</clipPath>
+</defs>
+</svg>
```

Comment:
Hey @FrancoMaxime,

We're glad we caught your eye. As for the icons, because of how the build process works, we have a pretty strict format, if you wish to contribute, make sure to read [our icon design guidelines](https://lucide.dev/docs/icon-design-guide) carefully.

There are several issues with the SVG provided:
- The icon should be on a 24 by 24 pixel grid
- No stroke, fill, transform attributes should be present on elements
- Filters and clip paths are unsupported
- The icon doesn't have the necessary [global attributes](https://lucide.dev/docs/icon-design-guide#global-attributes) 

Since I've already designed this icon as part of #950, I'll add that design as a suggestion instead of cleaning up the code, this is what an icon SVG file should generally look like:

```suggestion
<svg
  xmlns="http://www.w3.org/2000/svg"
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <path d="M15 10v4" />
  <path d="M6 18v.01" />
  <path d="M10 18v.01" />
  <rect x="2" y="14" width="20" height="8" rx="2" />
  <path d="M12.17 7.17a4 4 0 0 1 5.66 0" />
  <path d="M9.34 4.34a8 8 0 0 1 11.32 0" />
</svg>
```

Reply:
Hi @karsa-mistmere, thanks for your review. I'm amazed by your solution, its so clear. This is my first time playing with svg and figma. 
I'm busy this afternoon, but i will try this evening to clean this svg. And if i can't, i will push your solution. 

Thank you again for all,

Maxime

Reply:
Hi, 
I'm unable to do something as clean as you @karsa-mistmere  with figma. I will try to use Inkscape. So if you're not in a rush, i will play with inkscape this evening.

Reply:
Hey Maxime,

I actually made mine in Figma, but there's a catch: it does need some manual optimisation, although I've found that Inkscape is generally even more finicky.

My usual design process is as follows:
1) I draw the icon in Figma using 2px strokes only on a 24x24px frame, e.g.:
    https://www.figma.com/file/YRbgRWQ3pilLjZ7ByR5Da3/lucide%2Frouter?node-id=1%3A3&t=rnIgNj6l1Ycnup4b-1
2) I duplicate an existing icon file to clone its root element
3) I copy the frame from Figma as SVG
4) I paste that SVG into [SVGOMG](https://jakearchibald.github.io/svgomg/)
5) I view its markup and copy and paste just the paths into the root `<svg>` element of my target SVG file.
6) I remove any stroke and fill attributes from the paths manually

Reply:
Ok, this time i think i got something good. That's not exactly like you did with the ellipse (I couldn't find any other way) 
Here is my version if you want to look: https://www.figma.com/file/Kq6HKflFd3RdODP1HcN6qx/Untitled?node-id=3%3A4&t=JpH5qDm45Muow5OU-0

Thanks for the help 🙏